### DISCUSSION TITLE
How to create a custom page with HTML / CSS / JS

### DISCUSSION THREAD

**Discussion by User9511** [markshust/docker-magento#1097] [Q&A]
Hey Guys,

I've got a landing page created on pc using html / css / js.

I've got magento luma theme installed  and I've created a new blank page.

How can I got about adding my html / css / js to this page ? what is the best approach ?

Thanks!

**Comment by GNUXDAR** [comment]
**New Custom Module**

The best thing is for you to create a module with a controller that calls a template page, and in that module load your CSS and JS.

![image](https://github.com/markshust/docker-magento/assets/6179849/32dabfb1-af55-4fe9-914b-a28b79f21212)
